domain-services/ifix-master-data-service/src/main/java/org/egov/repository/ChartOfAccountRepository.java

```diff
@@ -1,37 +1,64 @@
+
 package org.egov.repository;
 
+import java.util.ArrayList;
+import java.util.List;
 
-import lombok.extern.slf4j.Slf4j;
-import org.egov.repository.queryBuilder.ChartOfAccountQueryBuilder;
+import org.egov.producer.Producer;
+import org.egov.repository.querybuilder.ChartOfAccountQueryBuilder;
+import org.egov.repository.rowmapper.COARowMapper;
 import org.egov.web.models.COASearchCriteria;
 import org.egov.web.models.ChartOfAccount;
 import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.data.mongodb.core.MongoTemplate;
-import org.springframework.data.mongodb.core.query.Query;
+import org.springframework.jdbc.core.JdbcTemplate;
 import org.springframework.stereotype.Repository;
+import org.springframework.util.CollectionUtils;
 import org.springframework.web.client.RestTemplate;
 
-import java.util.List;
+import lombok.extern.slf4j.Slf4j;
 
 @Repository
+
 @Slf4j
 public class ChartOfAccountRepository {
 
-    @Autowired
-    private MongoTemplate mongoTemplate;
+	@Autowired
+	private ChartOfAccountQueryBuilder coaQueryBuilder;
+
+	@Autowired
+	private RestTemplate restTemplate;
 
-    @Autowired
-    private ChartOfAccountQueryBuilder coaQueryBuilder;
+	@Autowired
+	private Producer producer;
 
-    @Autowired
-    private RestTemplate restTemplate;
+	@Autowired
+	private JdbcTemplate jdbcTemplate;
+	
+	@Autowired
+	private ChartOfAccountQueryBuilder chartOfAccountQueryBuilder;
+	
+	@Autowired
+	private COARowMapper coaRowMapper;
+	
+	// Ghanshyam
+	/*
+	 * public void save(ChartOfAccount chartOfAccount) {
+	 * mongoTemplate.save(chartOfAccount); }
+	 */
 
-    public void save(ChartOfAccount chartOfAccount) {
-        mongoTemplate.save(chartOfAccount);
-    }
+	public List<ChartOfAccount> search(COASearchCriteria searchCriteria) {
+		List<ChartOfAccount> caos = new ArrayList<>();
```

Comment:
Correct the typo

Reply:
minor change, to be taken up as part of code cleanup.